src/main/java/net/greatstart/controllers/ProjectRestController.java

```diff
@@ -0,0 +1,51 @@
+package net.greatstart.controllers;
+
+import net.greatstart.mappers.ProjectMapper;
+import net.greatstart.model.Project;
+import net.greatstart.services.CategoryService;
+import net.greatstart.services.ProjectService;
+import net.greatstart.services.UserService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/project")
+public class ProjectRestController {
+    private static final String REDIRECT_TO_PROJECTS = "redirect:/project/";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "REDIRECT_TO_PROJECTS" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)